code/obj/item/device/scanners.dm

```diff
@@ -232,6 +237,21 @@ that cannot be itched
 				icon_state = "fs_pinfar"
 		SPAWN(0.5 SECONDS)
 			.(T)
+	verb/set_phrase()
```

Comment:
`set_phrase`? Also I'd prefer this be an option through `attack_self()` rather than a verb. You could do something like

```
var/choice = input(user, "What do you want to do with [src]?", "Selection") as null|anything in list("Find record", "Print last scan")
switch(choice)
```